card/spec/lib/card/diff_spec.rb

```diff
@@ -33,111 +33,111 @@ def p_diff
   describe 'traffic light' do
     it 'is green for addition' do
       a = "a"
-      b = "a b" 
+      b = "a b"
       db = Card::Diff::DiffBuilder.new(a,b)
       expect(db.green?).to be_truthy
       expect(db.red?).to be_falsey
     end
     it 'is red for deletion' do
       a = "a"
-      b = "" 
+      b = ""
       db = Card::Diff::DiffBuilder.new(a,b)
       expect(db.green?).to be_falsey
       expect(db.red?).to be_truthy
     end
     it 'is green and red for change' do
       a = "a"
-      b = "b" 
+      b = "b"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.